processor/configuration/src/main/resources/schemas/actions/azure_eventhubs_sink_0.1.json

```diff
@@ -0,0 +1,45 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "azure_namespace_name",
+    "azure_eventhub_name",
+    "azure_shared_access_name",
+    "azure_shared_access_key"
+  ],
+  "properties": {
+    "azure_namespace_name" : {
+      "title" : "Eventhubs Namespace",
+      "description" : "The eventhubs namespace",
+      "type" : "string"
+    },
+    "azure_eventhub_name" : {
+      "title" : "Eventhubs Name",
+      "description" : "The eventhub name",
+      "type" : "string"
+    },
+    "azure_shared_access_name" : {
+      "title" : "Share Access Name",
+      "description" : "EventHubs SAS key name",
+      "type" : "string",
+      "x-group" : "credentials"
+    },
+    "azure_shared_access_key" : {
+      "title": "Share Access Key",
+      "x-group": "credentials",
+      "oneOf": [
+        {
+          "title": "Share Access Key",
+          "description": "The key for EventHubs SAS key name",
+          "type": "string",
+          "format": "password"
+        },
+        {
+          "description": "An opaque reference to the azure_shared_access_key",
+          "type": "object",
+          "properties": {}
+        }
+      ]
+    }
```

Comment:
we use two versions EventHubs and eventhubs, probably we should unify the name we use. Or do we refer by both to a different thing?

Reply:
wdym?

Reply:
I mean this small difference in naming
- "The eventhubs namespace",
- "EventHubs SAS key name"

What is your opinion of using 'eventhubs' vs 'EventHubs'